canadian-journal-economics-default.csl

```diff
@@ -0,0 +1,611 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" demote-non-dropping-particle="never" version="1.0" class="in-text" page-range-format="chicago">
+  <info>
+    <title>Canadian Journal of Economics: InText Default (Author-Date)</title>
+    <id>http://www.zotero.org/styles/canadian-journal-economics-default</id>
+    <link rel="self" href="http://www.zotero.org/styles/canadian-journal-economics-default"/>
+    <link rel="documentation" href="http://www.chicagomanualofstyle.org/tools_citationguide.html"/>
```

Comment:
Can you update this "documentation" link? http://economics.ca/cje/en/style.php is probably the best link.

And can you add a "template" link so we know which CSL style you based yours on? See https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link